FinSearchUnified/Tests/Bundle/SearchBundle/CriteriaRequestHandler/SdymCriteriaRequestHandlerTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use FinSearchUnified\Constants;
+use Shopware\Bundle\SearchBundle\Condition\SimpleCondition;
+use Shopware\Components\Test\Plugin\TestCase;
+use Shopware\Bundle\SearchBundle\Criteria;
+use Enlight_Controller_Request_RequestHttp as RequestHttp;
+use FinSearchUnified\Bundle\SearchBundle\CriteriaRequestHandler\SdymCriteriaRequestHandler;
+use Shopware\Tests\Functional\Bundle\StoreFrontBundle\Helper;
+use Shopware\Tests\Functional\Bundle\StoreFrontBundle\TestContext;
+
+class SdymCriteriaRequestHandlerTest extends TestCase
+{
+    /** @var RequestHttp */
+    public $request;
+    /** @var Criteria */
+    public $criteria;
+    /** @var TestContext */
+    public $context;
+    /** @var SdymCriteriaRequestHandler */
+    public $handler;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+
+        $this->criteria = new Criteria();
+        $this->context = $this->getContext();
```

Comment:
Use the following code to get a context instead of the `getContext` method:
```
$contextService = Shopware()->Container()->get('shopware_storefront.context_service');
$context = $contextService->getShopContext();
```